doc/install/install_python.rst

```diff
@@ -0,0 +1,103 @@
+.. include:: ../links.inc
+
+.. _install-python:
+
+Installing Python
+=================
+
+MNE-Python runs within Python, and depends on several other Python packages.
```

Comment:
"runs within Python" sounds strange – what information does this convey that is not already contained in the name "MNE-Python"? If you want to explicitly mention that MNE-Python is a Python package, I'd write "MNE-Python is a Python package...".

Reply:
this page is for people who have probably never used Python before.  If they're coming from MATLAB they may not have any exposure to the idea of "packages" (MATLAB has toolboxes, but they are often bundled / centrally managed / not installed by end-users).  So I don't think it's outlandish to explain "you need python, and several other python packages, in order for MNE-Python to work".  Saying "it's a python package" may not convey that to a naive user.  In light of that, do you have a different suggestion of how to re-word to sound less strange?

Reply:
Maybe "MNE-Python requires Python and several other Python packages"?

Reply:
Perhaps remove "other" to make it even clearer?